src/main/scala/devsearch.lookup/RawDB.scala

```diff
@@ -0,0 +1,22 @@
+package devsearch.lookup
+
+import reactivemongo.api.MongoDriver
+import reactivemongo.api.collections.default.BSONCollection
+import reactivemongo.bson.BSONDocument
+import reactivemongo.core.commands.RawCommand
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+object RawDB {
+  val DB_SERVER = "localhost"
+  val DB_NAME = "devsearch"
+
+  val driver = new MongoDriver
```

Comment:
I think that it's OK for now. We don't have to be too crazy about this :)
I was only concerned that in the cluster the master node has no database, so accessing this object would cause an error. But since the master doesn't access it and object initialization is lazy, it's OK.

Try in your REPL:
```
scala> object Foo { val x = {println("hey"); 10}; val y = 11 }
defined object Foo

scala> Foo.y
hey
res0: Int = 11
```